enrollment-server-onboarding-api-model/src/main/java/com/wultra/app/enrollmentserver/api/model/onboarding/response/ConfigurationResponse.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2025 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.enrollmentserver.api.model.onboarding.response;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+
+import java.util.List;
+
+/**
+ * Configuration response.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@Builder
+public record ConfigurationResponse(
```

Comment:
There is one missing information here, and that's "expected document count".

For example, I have 4 documents available - one required - how many should I scan? 2? 3?

If 3, that means 1 required and 2 non-required.

This information should be available too, otherwise, it needs to be hardcoded...